### DISCUSSION TITLE
How does Masked loss work?

### DISCUSSION THREAD

**Discussion by codecomputation** [bmaltais/kohya_ss#2897] [General]
It checked Masked loss in the UI and specify this additional parameters:
 --conditioning_data_dir "C:\SD\masks"

The masks seems to have an effect on training. But, the content in the black areas are learned regardless once a certain number of steps are reached. Is it because the mask doesn't fully eliminate learning in the blacked out areas?

**Comment by TotallyTroll** [comment]
I'm also curious where 'conditioning_data_dir' should be located relative to the training image location. When 'mask loss' parameter enabled.
Specifying --conditioning_data_dir "X:\some_folder" in toml config toml file seems to be not working. While this paramterer removed every time you save config in the UI.
So, I'm as well interested to know how to catually apply mask during training?
